docs/tutorials/text_sentiment_ngrams.ipynb

```diff
@@ -0,0 +1,419 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluate the model with test dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('Checking the results of test dataset...')\n",
+    "test_loss, test_acc = valid_test_func(test_dataset)\n",
+    "print(f'\\tLoss: {test_loss:.4f}(test)\\t|\\tAcc: {test_acc * 100:.1f}%(test)')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Checking the results of test dataset...\n",
+    "\n",
+    "        Loss: 0.0237(test)      |       Acc: 90.5%(test)\n",
+    "        \n",
+    "The results are consistent with the reference paper [Bag of Tricks for Efficient Text Classification](https://arxiv.org/abs/1607.01759)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test on a random news\n",
+    "\n",
+    "Use the best model so far and test a golf news."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import re\n",
+    "from torchtext.data.utils import ngrams_iterator\n",
+    "from torchtext.data.utils import get_tokenizer\n",
+    "\n",
+    "ag_news_label = {1 : \"World\",\n",
+    "                 2 : \"Sports\",\n",
+    "                 3 : \"Business\",\n",
+    "                 4 : \"Sci/Tec\"}\n",
+    "\n",
+    "def predict(text, model, dictionary, ngrams):\n",
+    "    tokenizer = get_tokenizer(\"basic_english\")\n",
+    "    with torch.no_grad():\n",
+    "        text = torch.tensor([dictionary[token]\n",
+    "                            for token in ngrams_iterator(tokenizer(text), ngrams)])\n",
+    "        output = model(text, torch.tensor([0]))\n",
+    "        return output.argmax(1).item() + 1\n",
+    "\n",
+    "example_text_string = \"MEMPHIS, Tenn. – Four days ago, Jon Rahm was \\\n",
+    "    enduring the season’s worst weather conditions on Sunday at The \\\n",
+    "    Open on his way to a closing 75 at Royal Portrush, which \\\n",
+    "    considering the wind and the rain was a respectable showing. \\\n",
+    "    Thursday’s first round at the WGC-FedEx St. Jude Invitational \\\n",
+    "    was another story. With temperatures in the mid-80s and hardly any \\\n",
+    "    wind, the Spaniard was 13 strokes better in a flawless round. \\\n",
+    "    Thanks to his best putting performance on the PGA Tour, Rahm \\\n",
+    "    finished with an 8-under 62 for a three-stroke lead, which \\\n",
+    "    was even more impressive considering he’d never played the \\\n",
+    "    front nine at TPC Southwind.\"\n",
+    "\n",
+    "new_label = \"\"\n",
+    "vocab = train_dataset.get_vocab()\n",
+    "with open(\"text_classification.pt\", 'rb') as f:\n",
+    "    model = torch.load(f).to(\"cpu\")\n",
+    "    news_label = ag_news_label[predict(example_text_string,\n",
```

Comment:
If you give "example_text_string" a smaller name (maybe "example") then you can merge this line with the print statement and make this a bit shorter.